drivers/pcie/host/pcie.c

```diff
@@ -291,8 +303,17 @@ unsigned int pcie_alloc_irq(pcie_bdf_t bdf)
 	    irq >= CONFIG_MAX_IRQ_LINES ||
 	    arch_irq_is_used(irq)) {
 
+	/*
+	 * In some platforms, PCI interrupts are hardwired to specific interrupt inputs
```

Comment:
Minor: I think the consistent code comment style is to begin the text right after `/* ...` instead of the next line.